pkg/yurthub/server/server.go

```diff
@@ -38,15 +47,17 @@ type Server interface {
 // and hubServer handles requests by hub agent itself, like profiling, metrics, healthz
 // and proxyServer does not handle requests locally and proxy requests to kube-apiserver
 type yurtHubServer struct {
-	hubServer        *http.Server
-	proxyServer      *http.Server
-	dummyProxyServer *http.Server
+	hubServer              *http.Server
+	proxyServer            *http.Server
+	secureProxyServer      *http.Server
+	dummyProxyServer       *http.Server
+	dummySecureProxyServer *http.Server
 }
 
 // NewYurtHubServer creates a Server object
 func NewYurtHubServer(cfg *config.YurtHubConfiguration,
 	certificateMgr interfaces.YurtCertificateManager,
-	proxyHandler http.Handler) (Server, error) {
+	proxyHandler http.Handler, stopCh <-chan struct{}) (Server, error) {
```

Comment:
stopCh is not used?

Reply:
The latest code has been commited. stopCh parameter has been removed.